src/langcheck/metrics/de/reference_based_text_quality.py

```diff
@@ -0,0 +1,343 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional
+
+import torch
+from openai import OpenAI
+from rouge_score import rouge_scorer
+from rouge_score.tokenizers import Tokenizer
+from sentence_transformers import SentenceTransformer, util
+
+from langcheck.metrics._validation import validate_parameters_reference_based
+from langcheck.metrics.de._tokenizers import DeTokenizer
+from langcheck.metrics.en.reference_based_text_quality import \
+    semantic_similarity as en_semantic_similarity
+from langcheck.metrics.metric_value import MetricValue
+from langcheck.utils.progess_bar import tqdm_wrapper
+
+MODEL_NAME = "sentence-transformers/distiluse-base-multilingual-cased-v1"
```

Comment:
moved to inside the function, like in JA. I felt like keeping it on top for easy changes could be useful, but I suspect this would be for a general "multilang" rewrite